tests/cupy_tests/manipulation_tests/test_join.py

```diff
@@ -173,10 +173,22 @@ def test_stack_value(self):
 
     @testing.with_requires('numpy>=1.10')
     @testing.numpy_cupy_array_equal()
-    def test_stack_with_axis(self, xp):
+    def test_stack_with_axis1(self, xp):
         a = testing.shaped_arange((2, 3), xp)
         return xp.stack((a, a), axis=1)
 
+    @testing.with_requires('numpy>=1.10')
+    @testing.numpy_cupy_array_equal()
+    def test_stack_with_axis2(self, xp):
+        a = testing.shaped_arange((2, 3), xp)
+        return xp.stack((a, a), axis=2)
+
+    @testing.with_requires('numpy>=1.13')
```

Comment:
It is better to write `try-except`manually because `numpy<1.13` behaves actually **differently**.